src/main/webapp/instructorHelp.jsp

```diff
@@ -5329,7 +5113,7 @@
                                               </ul>
                                           </div>
                                           <div class="feedback-path-others margin-top-7px" style="display:none;">
-                                              <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">  
+                                              <div data-original-title="Who will give feedback" class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px" data-toggle="tooltip" data-placement="top" title="">
```

Comment:
Sorry about these extraneous changes. The way I was editing the document must have modified the spacing. I will make the changes soon.